opendbc/car/toyota/carstate.py

```diff
@@ -79,7 +82,19 @@ def update(self, cp, cp_cam, *_) -> structs.CarState:
     ret.brakePressed = cp.vl["BRAKE_MODULE"]["BRAKE_PRESSED"] != 0
     ret.brakeHoldActive = cp.vl["ESP_CONTROL"]["BRAKE_HOLD_ACTIVE"] == 1
 
-    ret.gasPressed = cp.vl["PCM_CRUISE"]["GAS_RELEASED"] == 0
+    if self.CP.flags & ToyotaFlags.SECOC.value:
+      self.secoc_synchronization = copy.copy(cp.vl["SECOC_SYNCHRONIZATION"])
+      ret.gas = cp.vl["GAS_PEDAL"]["GAS_PEDAL_USER"]
```

Comment:
`ret.gas` seems to have been removed for the other cars, so probably can go here as well.

Reply:
Not sure if it was removed, or just never added for Toyota, I'd have to walk through git blame. However, every other car port is setting `ret.gas`. ISTM we shouldn't remove it for SecOC, but (in a separate PR) add it for non-SecOC.

Reply:
I think it was removed here https://github.com/commaai/openpilot/pull/31903/files#diff-94720bc1c4f1ce01d86ce63b3c8278d1a201cf98e0796d5db9f1c0081d5a8a15.

The `ret.gas` still being present for the RAV4 prime is probably because my PR predates that commit.

Reply:
Looks like it was lost in commaai/openpilot#30194, and it's still desirable to put it back someday once the right logic is found. commaCarDatasets actually makes that a lot easier.

I think it makes sense to keep it in this port.